src/ProjectTemplates/Microsoft.Extensions.AI.Templates/src/ChatWithCustomData/ChatWithCustomData-CSharp.Web/Program.Aspire.cs

```diff
@@ -41,15 +36,13 @@
 
 #if (UseAzureAISearch)
 builder.AddAzureSearchClient("azureAISearch");
-
-builder.Services.AddSingleton<IVectorStore, AzureAISearchVectorStore>();
+builder.Services.AddAzureAISearchVectorStore();
 #elif (UseQdrant)
 builder.AddQdrantClient("vectordb");
-
-builder.Services.AddSingleton<IVectorStore, QdrantVectorStore>();
+builder.Services.AddQdrantVectorStore();
 #else // UseLocalVectorStore
-var vectorStore = new JsonVectorStore(Path.Combine(AppContext.BaseDirectory, "vector-store"));
-builder.Services.AddSingleton<IVectorStore>(vectorStore);
+var vectorStorePath = Path.Combine(AppContext.BaseDirectory, "vector-store.db");
+builder.Services.AddSqliteVectorStore($"Data Source={vectorStorePath}");
```

Comment:
It's generally better to just register VectorStoreCollections directly in DI, rather than registering a VectorStore and then calling GetCollection() on it each time (it's both more ceremony and worse for perf, since GetCollection() isn't free). Since this template seems to use more than one collection, using DI keyed services seems like a good match.

Unfortunately, we're just now doing last changes to the DI registration - https://github.com/microsoft/semantic-kernel/pull/12043 (yes, we're working right to the last minute...). Once this is done, you should be able to easily and cleanly call `AddSqliteCollection("Chunks", ...)` and then simply resolve that.